ffi/derive/src/impl_visitor.rs

```diff
@@ -131,18 +134,30 @@ impl<'ast> ImplDescriptor<'ast> {
 }
 
 impl FnDescriptor {
-    fn from_impl_method(self_ty: &syn::Path, node: &syn::ImplItemMethod) -> Self {
-        let mut visitor = FnVisitor::new(Some(self_ty));
+    pub(crate) fn from_impl_method(
```

Comment:
```suggestion
    pub fn from_impl_method(
```

**Explanation:** This function is defined in a private module in a proc-macro crate. If you wanted to import it, the only place where you would be able to do so is yet another proc-macro crate.

Reply:
Thank you for explanation!